components/playlist/browser/resources/api/api.ts

```diff
@@ -67,29 +67,32 @@ class API {
     )
   }
 
-  addMediaFilesFromActiveTabToPlaylist (playlistId: string) {
+  addMediaFilesFromActiveTabToPlaylist(playlistId: string) {
     this.#pageHandler.addMediaFilesFromActiveTabToPlaylist(
       playlistId,
       /* canCache */ true
     )
   }
 
-  moveItemFromPlaylist (playlistId: string, itemId: string[]) {
+  moveItemFromPlaylist(playlistId: string, itemId: string[]) {
     this.#nativeUI.showMoveItemsUI(playlistId, itemId)
   }
 
-  removeItemFromPlaylist (playlistId: string, itemId: string) {
+  removeItemFromPlaylist(playlistId: string, itemId: string) {
     this.#pageHandler.removeItemFromPlaylist(playlistId, itemId)
   }
 
-  recoverLocalData (playlistItemId: string) {
+  recoverLocalData(
+    playlistItemId: string,
+    updateMediaSrcBeforeRecovery: boolean = false
```

Comment:
optional: boolean is implicit
```suggestion
    updateMediaSrcBeforeRecovery = false
```